frequency: 50
two_d_mode: true
publish_tf: true

map_frame: map
odom_frame: odom
base_link_frame: base_link
world_frame: odom

# âœ… RELATIVO ao namespace /car_like
odom0: odom

odom0_config: [true,  true,  false,
               false, false, true,
               true,  true,  false,
               false, false, true,
               false, false, false]
odom0_differential: false
odom0_relative: false

